src/sonarwhal-theme/scripts/link-filter.js

```diff
@@ -25,10 +25,12 @@ hexo.extend.filter.register('before_post_render', (data) => {
             matchHtmlUrl = matchHtmlUrl.replace(/\((.*?)\)/g, '(../$1)');
         }
 
-        // ../../../../rule-axe/README/ => rule-axe
+        // ../../../../rule-axe/README/ => rule-axe/
         const matchRuleUrl = matchHtmlUrl.match(ruleUrlRegex);
         const newUrl = isRulePage && matchRuleUrl ? `(${matchRuleUrl.pop()}/)` : matchHtmlUrl;
 
         data.content = data.content.replace(matchMdUrl, newUrl);
+        // Offset `lastIndex` due to the length change after string replacement.
```

Comment:
Can you provide some examples in the comments? Thanks!

Reply:
@alrra Done